saleor/order/views.py

```diff
@@ -112,3 +112,26 @@ def cancel_payment(request, order):
             form.save()
         return redirect('order:payment', token=order.token)
     return HttpResponseForbidden()
+
+
+def create_password(request, token):
+    if request.user.is_authenticated():
+        return redirect('order:payment', token=token)
```

Comment:
Please redirect to `order:details` page instead of order payment - this order is already paid.